maas-schemas/schemas/maas-backend/customers/customer.json

```diff
@@ -61,8 +61,14 @@
       "items": {
         "$ref": "http://maasglobal.com/core/components/common.json#/definitions/agencyId"
       }
+    },
+    "virtualCards": {
+      "type": "array",
+      "items": {
+        "$ref": "http://maasglobal.com/maas-backend/customers/virtual-cards/virtualCard.json"
+      }
     }
   },
-  "required": ["personalData", "paymentSources", "balances", "region", "authorizations"],
+  "required": ["personalData", "paymentSources", "balances", "region", "authorizations", "virtualCards"],
```

Comment:
Relating to the backend forward compat also, would it be easier if virtualCards is not required and we don't need the forward compat at all?

Reply:
Because virtualCards will become a key part of the London plan, and it relates to features dealing with financial transactions, I think it's better to be rather more strict than loose.